lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4161,6 +4203,10 @@ namespace {
       if (correctSwiftName)
         markAsVariant(result, *correctSwiftName);
 
+      // For constexpr vars, we can create an accessor with a numeric literal.
```

Comment:
Hoisting out the check for if the value is `constexpr` would be better IMO.  It avoids the need for the comment to explain what is happening, and is quite concise:

```c++
if (decl->isConstexpr())
  if (auto accessor = tryCreateConstexprAccessor(decl, result))
    result->setAccessors(SourceLoc(), {accessor}, SourceLoc());
```